src/core/public/chrome/README.md

```diff
@@ -0,0 +1,127 @@
+
+##    Chrome Service
+
+- [About!](#about-)
+- [Nav Controls Service](#navcontrolsservice-)
+- [Nav Links Service](#navlinksservice-)
+- [Recently Accessed Service](#recentlyaccessedservice-)
+- [Doc Title Service](#doctitleservice-)
+- [UI](#ui-)
+
+## About :
+- **Signature** - `export interface ChromeStart`
+
+The chrome service is a high level UI service that is part of CoreStart (Core services exposed to the Plugin start lifecycle) and offers other plugins a way to add navigation controls to the UI, edit the document title, manipuate navlinks on global header as well as edit the Recent accessed tab. it consists of these sub-services/components.
+- NavControlsService : for registering new controls to be displayed in the navigation bar.
+- NavLinksService : for manipulating nav links.
+- RecentlyAccessedService : for recently accessed history.
+- DocTitleService: for accessing and updating the document title
+- UI : All the UI components,icons for e.g  header, loaders.
+
+
+- How to access ? add in interface `chrome: ChromeStart && chrome.servicesName => e.g chrome.docTitle.method`
+- Where it is getting Registered/Executed [Staring Point](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/2.1/src/core/public/rendering/rendering_service.tsx)
+- How header component is getting rendered `const chromeUi = chrome.getHeaderComponent(); `
+- Chrome Methods </b> [See chrome interface/methods](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/2.1/src/core/public/chrome/chrome_service.tsx)
+
+
+### NavControlsService : 
+- Interface : ChromeNavControls
+- **Signature** - `navControls: ChromeNavControls`
+- Registering new controller 
+
+Example : 
+Register a left-side nav control rendered with React.
+
+```jsx
+chrome.navControls.registerLeft({
+  mount(targetDomElement) {
+    ReactDOM.mount(<MyControl />, targetDomElement);
+    return () => ReactDOM.unmountComponentAtNode(targetDomElement);
+  }
+})
+```
+### NavLinksService : 
+- Interface :  ChromeNavLinks
+- **Signature** - `navLinks: ChromeNavLinks`
+- e.g Method : 
+
+Get an observable for a sorted list of navlinks :-
```

Comment:
Can we make formats more consistent? For example, can we format these two parts same?

<img width="1193" alt="Screen Shot 2022-07-15 at 15 14 02" src="https://user-images.githubusercontent.com/79961084/179318266-b08a5c15-3929-4af0-9bdd-e9ee3876230c.png">